### PR TITLE
feat: [SRM-11330]: fixing whitespacing issues

### PR DESCRIPTION
**Pull request by harshilbits**
#### Screenshots
Before:
New Note
After:
New Note

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10388)

File: src/modules/85-cv/pages/slos/components/CVCreateSLO/CVCreateSLO.module.scss [hunk 1]
```diff
@@ -6,7 +6,6 @@
  */
 
 .pageBody {
-  max-height: calc(100vh - 120px);
   padding: var(--spacing-xlarge) !important;
```
Comment (abhinavrastogi-harness) [inline src/modules/85-cv/pages/slos/components/CVCreateSLO/CVCreateSLO.module.scss:9]: do we need custom padding? or can we remove it too?

**Review by abhinavrastogi-harness — APPROVED** [review]

File: src/modules/85-cv/pages/slos/components/CVCreateSLO/CVCreateSLO.module.scss (hunk 1 above)
Comment (harshilbits) [reply src/modules/85-cv/pages/slos/components/CVCreateSLO/CVCreateSLO.module.scss:9]: I tested this , padding was actually required.

**Comment by harshilbits** [comment]
retrigger titlecheck

**Review by deepesh-ui — APPROVED** [review]

**Pull request merged**